src/app/examples/todos/todos.reducer.spec.ts

```diff
@@ -0,0 +1,60 @@
+import * as fromTodo from './todos.reducer';
+
+describe('TodoReducer', () => {
+  describe('undefined action', () => {
```

Comment:
Hi @shootermv ! Can we please remove nested describe blocks? Every action has anyway only one test.